### PR TITLE
Fixed button font sizes

### PR DESCRIPTION
**Pull request by DonaldKLee**

### CODE REVIEW

**Review by meleongg — APPROVED** [review]
LGTM (but wait for Amy to approve before merging)

**Review by ajo01 — COMMENTED** [review]
I checked out the branch and it looks like p.size is undefined right now so the styles aren't changing as expected. This is because the size value from the props isn't being passed to the Button div. To do so, you can add `size={props.size}` in `<Button/>`

**Pull request merged**
